src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -156,7 +160,7 @@ class DocumentDetails extends Component {
     const { searchQuery } = this.state;
     return (
       <>
-        <Overridable id="DocumentDetails.top">
+        <Overridable id="DocumentDetails.top" searchQuery={searchQuery}>
```

Comment:
This is for sure needed, but not enough: there are also these 2 functions `this.onSearchInputChange` and `this.onSearchClick` that are not callable when overriding.
The interface when overriding should be:
```
(currentQueryString, onInputChange, executeSearch) => {}
```
which probably means:
```
<Overridable id="DocumentDetails.top" currentQueryString={searchQuery} onInputChange={this.onSearchInputChange} executeSearch={this.onSearchClick}>
...
              <SearchBar
                currentQueryString={currentQueryString}
                onInputChange={onInputChange}
                executeSearch={executeSearch}
                placeholder="Search for books..."
                className="document-details-search-bar"
              />
...
```
to be tested...

It would be nice also to change `"Search for books..."` to `"Search for literatures..."`

Reply:
I have tested this (with a custom overridable component)

- I have updated the Overridable components with explicit props
- I have updated the placeholders
- removed the unused class `document-details-search-bar` 
- renamed the style class to `literature` instead of documents since its used in series too.